src/sidebar/store/modules/groups.js

```diff
@@ -206,7 +206,11 @@ const getCurrentlyViewingGroups = createSelector(
  *
  * // Selectors
  * @prop {() => Group[]} allGroups
- * // TODO: add rest ...
+ * @prop {() => Group|undefined|null} focusedGroup
```

Comment:
Separate from this PR, we should fix the issue where `focusedGroup` and `getGroup` can return different falsey values. In practice it doesn't matter that much, but it still creates a small possibility of a bug.

Reply:
Ya, I noticed that, there wa a mis-match there.